examples/importing/catalog-contents/catalog-sharing/terraform.tfvars.example

```diff
@@ -0,0 +1,19 @@
+vcd_url      = "https://some-vcd.com"
+
+# organization where we create the new catalog
+org          = "myorg"
+
+# name of the new catalog
+catalog      = "mycatalog"
+
+# where to find the .OVA and .ISO to create catalog contents.
+# The directory "./test-resources" is at the top of this repository
+ova_path     = "../../../test-resources/"
```

Comment:
Not sure if just me, but I had to add an extra `..`, as executing `terraform apply` inside `examples/importing/catalog-contents/catalog-sharing` makes it search for the ISO file in `examples/test-resources/`.

```
Error: error uploading new catalog media: stat /terraform-provider-vcd/examples/test-resources/test.iso: no such file or directory
```
As this is more of an example it is not really important I guess, but more of a nitpick.

Reply:
Thanks. I copied the value from the vApp example, which is one lever higher.
Fixed